.github/workflows/main.yml

```diff
@@ -38,13 +38,12 @@ jobs:
       - name: "Run tox targets for ${{ matrix.python-version }}"
         run: "python -m tox"
 
-      # - name: "Report to coveralls"
+      - name: "Report to coveralls"
       #   # coverage is only created in the py39 environment
       #   # --service=github is a workaround for bug
       #   # https://github.com/coveralls-clients/coveralls-python/issues/251
-      #   if: "matrix.python-version == '3.9'"
```

Comment:
coverage data is only generated in the 3.9 environment

Reply:
got it